src/shared/services/window/window.service.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { WindowService } from './window.service';
+
+describe('WindowService', () => {
+
+    let windowService: WindowService = new WindowService();
```

Comment:
Indenting of 4 spaces in here. Should be 2.

Reply:
Done